xbmc/pvr/dialogs/GUIDialogPVRGroupManager.cpp

```diff
@@ -192,7 +192,7 @@ bool CGUIDialogPVRGroupManager::ActionButtonUngroupedChannels(CGUIMessage &messa
     {
       if (m_channelGroups->GetFolderCount() == 0)
       {
-        CGUIDialogOK::ShowAndGetInput(19033,19137,0,19138);
+        CGUIDialogOK::ShowAndGetInput(19033,19137);
```

Comment:
There are so many missing finding them all will be a royal pain, and eventually inevitable if cleaning up unused strings and exact duplicates is in any way on a todo list.

Once string fixes are done, I was planing on getting that ball rolling. Not looking forward to that.